.github/self_review.md

```diff
@@ -0,0 +1,14 @@
+## Self review
+
+You should always review your own PR first.
+
+For changes, make sure that you:
+
+- [ ] Confirm the changes you made adheres to [contribution guide](../../CONTRIBUTING.md).
```

Comment:
Is CONTRIBUTING.md file location correct here? When I try to access this file, I get a `File not found` page.